cirq-core/cirq/qis/measures.py

```diff
@@ -273,3 +273,28 @@ def von_neumann_entropy(
     if validate:
         _ = quantum_state(state, qid_shape=qid_shape, copy=False, validate=True, atol=atol)
     return 0.0
+
+
+def entanglement_fidelity(operation: 'cirq.SupportsChannel') -> float:
+    r"""Returns entanglement fidelity of a given quantum channel.
+
+    Entanglement fidelity $F_e$ of a quantum channel $E: L(H) \to L(H)$ is the overlap between
+    the maximally entangled state $|\phi\rangle = \frac{1}{\sqrt{dim H}} \sum_i|i\rangle|i\rangle$
+    and the state obtained by sending one half of $|\phi\rangle$ through the channel $E$, i.e.
+
+        $$
+        F_e = \langle\phi|(E \otimes I)(|\phi\rangle\langle\phi|)|\phi\rangle
+        $$
+
+    where $I: L(H) \to L(H)$ is the identity map.
+
+    Args:
+        operation: Quantum channel whose entanglement fidelity is to be computed.
+    Returns:
+        Entanglement fidelity of the channel represented by operation.
+    """
+    f = 0.0
+    for k in protocols.channel(operation):
+        f += np.abs(np.trace(k)) ** 2
+    n_qubits = protocols.num_qubits(operation)
+    return float(f / 4 ** n_qubits)
```

Comment:
Could you add a test for an operation with more than one qubit?

Reply:
Good point. Added 2- and 3-qubit test cases for unitary and depolarizing channels.